config/cmake/addlibzmq.cmake

```diff
@@ -159,11 +159,21 @@ endif()
 if(HELICS_BUILD_CXX_SHARED_LIB OR NOT HELICS_DISABLE_C_SHARED_LIB)
 
 	if(NOT HELICS_USE_ZMQ_STATIC_LIBRARY)
-		install(
-			FILES $<TARGET_FILE:${zmq_target_output}>
-			DESTINATION ${CMAKE_INSTALL_BINDIR}
-			COMPONENT libs
-		)
+		set_target_properties(${zmq_target_output} PROPERTIES PUBLIC_HEADER "")
+		if(NOT ${CMAKE_VERSION} VERSION_LESS "3.13")
+		   install(TARGETS ${zmq_target_output}
+               RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+               ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+               LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+               FRAMEWORK DESTINATION "Library/Frameworks"
+		   )
+		elseif(WIN32)
+		   install(
+               FILES $<TARGET_FILE:${zmq_target_output}>
+               DESTINATION ${CMAKE_INSTALL_BINDIR}
+               COMPONENT libs
+		   )
+		endif()
```

Comment:
Is there anything we can put in an additional else() clause that would be useful and correct?

Reply:
The only thing would be an install with the TARGET_FILE for zmq, and then trying to create the correct symlinks manually. I think that would be more trouble to get working and maintain than it is worth.

Maybe an error message saying "upgrade to CMake 3.13+ or enable HELICS_USE_ZMQ_STATIC_LIBRARY=ON".